docs/source/developing/writing-docs.adoc

```diff
@@ -56,7 +56,7 @@ In this case you need to build the container with a different user id.
 This will ensure that the _docs_ user in the container will be able to create files and directories.
 
 ----
-$ make build_docs_container IMAGE_UID=$(id -u)
+$ make gen_docs IMAGE_UID=$(id -u)
```

Comment:
To be honest, I doubt that this is making any difference. It is the _build_docs_container_ target which takes the IMAGE_UID parameter. gen_docs does not use it. Once you have the container build with the right UID, you should be able to run gen_docs without specifying IMAGE_UID. 

Any chance that you tried to run _make build_docs_container IMAGE_UID=$(id -u)_ and you already had a built image. I see a problem here. If you have already an image and run the _build_docs_container_ target, it won't actually rebuild, since there are no changes to Dockerfile. The build will reuse the cached layers. 

IMO what's missing is a note that one needs to first clean the Docker cache by removing the old image. 

Alternatively we could either have an optional _--no-cache_ option for the _build_docs_container_ or we offer a target to remove the cached image. The latter makes probably most sense. In the case someone sees the permission errors, he needs to first delete the old image and then rebuild the container using the IMAGE_UID parameter. 

@budhrg, any chance that at some stage during your experiments you cleared/deleted the old image. As said, `make gen_docs IMAGE_UID=$(id -u)` does not seem to me to be an actual fix.